android/.buildconfig.yaml

```diff
@@ -2,3 +2,7 @@ libraryVersion: 0.1.0
 groupId: uniffi
 artifactId: nimbus
 description: A uniffi generated android bindings for the Nimbus SDK
+libUrl: https://github.com/mozilla/glean
```

Comment:
Is it correct to reference glean here?

Reply:
No, this is copy-pasta, will clean it up!